doc/classes/TextEdit.xml

```diff
@@ -1111,7 +1111,7 @@
 			<param index="1" name="gutter" type="int" />
 			<param index="2" name="clickable" type="bool" />
 			<description>
-				If [param clickable] is [code]true[/code], makes the [param gutter] on [param line] clickable. See [signal gutter_clicked].
+				If [param clickable] is [code]true[/code], makes the [param gutter] on [param line] clickable. This is like [method set_gutter_clickable], but for a single line. If [method is_gutter_clickable] is [code]true[/code], this will not have an effect. See [method is_line_gutter_clickable] and [signal gutter_clicked].
```

Comment:
I'm a little confused on when to use the param tag. Should we make it consistent so every `given line` becomes `given [param line]`, or is it fine as is?

Reply:
I'd personally use `[param]` tags every time, but sometimes they make localization more laborious as they cannot be translated. I think in this case it looks fine.
There's unfortunately no general rules or consistency about it, and some descriptions miss `[param]` just because they were written before the tag was introduced.

I have this in my own notes:
> - Avoid using the [param] tag as a replacement for an important subject of the sentence. Ideally, rephrase the sentence as if the parameter names were generic, single-letter words. This makes descriptions easier to localize, while still keeping the [param] tags. Note that parameter names remain unlocalized.